10/alpine/Dockerfile

```diff
@@ -16,14 +16,14 @@ RUN addgroup -g 1000 node \
           ;; \
         *) ;; \
       esac \
-  && ( \
+  && if [ ! -z "${CHECKSUM}" ]; then \
```

Comment:
`! -z` is just `-n` if you wanted, but it looks like it will all work correctly.  :+1: